README.md

```diff
@@ -1,4 +1,15 @@
-# osbuilder [![Build Status](https://travis-ci.org/kata-containers/osbuilder.svg?branch=master)](https://travis-ci.org/kata-containers/osbuilder) 
+[![Build Status](https://travis-ci.org/kata-containers/osbuilder.svg?branch=master)](https://travis-ci.org/kata-containers/osbuilder)
+
+# osbuilder
+
+* [Introduction](#introduction)
+* [Terms](#terms)
+* [Usage](#usage)
+    * [rootfs creation](#rootfs-creation)
+    * [Image creation](#image-creation)
+    * [Initrd creation](#initrd-creation)
```

Comment:
Uncertain on this - should either "rootfs" be capitalized here or "Initrd" be lower case for consistency?

Reply:
Updated to capitalise `Rootfs`.